docs/install/install.rst

```diff
@@ -1,51 +1,71 @@
 .. meta::
-  :description: TransferBench documentation 
+  :description: TransferBench documentation
   :keywords: TransferBench, API, ROCm, HIP
 
+.. _install-transferbench:
+
 ---------------------------
-TransferBench installation
+Installation
 ---------------------------
 
-The following software is required to install TransferBench:
+This document provides information required to install and build TransferBench.
 
-* ROCm stack installed on the system (HIP runtime)
-* `libnuma` installed on the system
+Prerequisite
+---------------
 
---------------------------
-Building TransferBench
---------------------------
+* Install ROCm stack on the system (HIP runtime)
```

Comment:
Add something like: For more information, see HIP. 

This line is hard to understand. Do you need both ROCm and the HIP component installed?

Reply:
HIP runtime is installed when u install ROCm. Changed it to "Install ROCm stack on the system to obtain HIP runtime" to make this explicit.